File: base_module/src/main/java/kt/module/base_module/data/RvDataSection.java
package kt.module.base_module.data;

import com.chad.library.adapter.base.entity.SectionEntity;

public class RvDataSection extends SectionEntity<RvData> {
    public RvDataSection(boolean isHeader, String header) {
        super(isHeader, header);
    }

    public RvDataSection(RvData rvData) {
        super(rvData);
    }
}

File: base_module/src/main/java/kt/module/base_module/data/RvDataSection.kt
package kt.module.base_module.data

import com.chad.library.adapter.base.entity.SectionEntity

class RvDataSection : SectionEntity<RvData> {
    constructor(isHeader: Boolean, header: String) : super(isHeader, header)

    constructor(rvData: RvData) : super(rvData)
}